internal/compiler/lookup.rs

```diff
@@ -1154,6 +1155,7 @@ impl<'a> LookupObject for NumberExpression<'a> {
             .or_else(|| f("asin", member_function(BuiltinFunction::ASin)))
             .or_else(|| f("acos", member_function(BuiltinFunction::ACos)))
             .or_else(|| f("atan", member_function(BuiltinFunction::ATan)))
+            .or_else(|| f("atan2", member_function(BuiltinFunction::ATan2)))
```

Comment:
this one line adds support for something like `x.atan2(y)` in Slint. 
I know that Rust support the same syntax, but i find it a bit weird in that case.
What do you think?